src/Adapters/EventSource.Net46/EventSourceListeningRequest.cs

```diff
@@ -0,0 +1,72 @@
+﻿// -----------------------------------------------------------------------
```

Comment:
OK, removed. 

I added them in the first place because I saw some headers in some files in one of the AI repos.